app/build.gradle

```diff
@@ -50,7 +50,7 @@ dependencies {
     implementation 'com.google.android.material:material:1.6.1'
 
     // WebRTC
-    implementation 'org.webrtc:google-webrtc:1.0.+'
+    implementation 'org.webrtc:google-webrtc:1.0.32006'
```

Comment:
This fixes the compiler error due to failing to resolve webrtc maven-metadata 

<img width="958" alt="image" src="https://github.com/user-attachments/assets/af687b09-256b-4c7f-8ee6-329c5100c05a">